stdlib/Printf/src/Printf.jl

```diff
@@ -762,6 +849,48 @@ const UNROLL_UPTO = 16
     return pos
 end
 
+
+@inline function findonesplace(buf, lbound, rbound)
+    decimalpoint = findlast(==(UInt8('.')), buf[lbound:rbound])
```

Comment:
We should use a `view` here instead of `getindex` which makes a copy

Reply:
Replaced as
```
decimalpoint = findlast(==(UInt8('.')), view(buf, lbound:rbound))
```